Tools/ros2/ardupilot_sitl/CMakeLists.txt

```diff
@@ -24,12 +24,12 @@ message(STATUS "WAF_COMMAND: ${WAF_COMMAND}")
 set(WAF_CONFIG $<$<OR:$<CONFIG:Debug>,$<CONFIG:RelWithDebInfo>>:"--debug">)
 
 add_custom_target(ardupilot_configure ALL
-  ${WAF_COMMAND} configure --board sitl --enable-dds ${WAF_CONFIG}
+  ${WAF_COMMAND} configure --board sitl --enable-dds --enable-networking ${WAF_CONFIG}
```

Comment:
Can you add a note about the use of `--enable-networking` in the docs. This flag is not currently set in master but AP_DDS in SITL is working with UDP, so it's not clear when it's required.

Reply:
Since it's about to disappear, I'll leave it out. If that other PR doesn't go in within a week, I'll add some docs.

Reply:
Resolved, the flag no longer exists.